File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/Precios.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class Precios
    <Key>
    Public Property idPrecio As Integer
    Public Property idMenu As Integer
    <ForeignKey("idMenu")>
    Public Property Menu As Menu
    Public Property precio As Single
    Public Property activo As Boolean
End Class

File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/Precios.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class Precios
    {
        [Key]
        public int idPrecio { get; set; }

        public int idMenu { get; set; }
        [ForeignKey("idMenu")]
        public Menu Menu { get; set; }

        public float precio { get; set; }

        public bool activo { get; set; }
    }
}
